src/Viewer/Viewer.js

```diff
@@ -231,6 +232,24 @@ export function Viewer ({fileInfo, prettifyLog, logEventNumber, timestamp}) {
         modifyFileMetadata(fileMetadata, logFileState.logEventIdx);
     }, [fileMetadata]);
 
+    /**
+     * Gets called BEFORE the Monaco Editor is mounted
+     */
+    const handleMonacoBeforeMount = () => {
+        // Disable the cached "pageSize" in case it causes a client OOM. If it
+        // doesn't, the saved value will be restored in handleMonacoOnMount().
```

Comment:
```suggestion
    /**
     * Unsets the cached page size in case it causes a client OOM. If it
     * doesn't, the saved value will be restored when restoreCachedPageSize is
     * called.
     */
    const unsetCachedPageSize = () => {
```

Reply:
I feel like naming the function after what it does is clearer when reading the code and seeing it used as the callback passed to `onBeforeMount`. Same with `onMount`.